src/test/resources/scripts/init.sql

```diff
@@ -88,9 +88,9 @@ CREATE TABLE IF NOT EXISTS FileMetadata (
     ADD FOREIGN KEY (FileId)
     REFERENCES File(FileId);
 
-ALTER TABLE FileMetadata
-    ADD FOREIGN KEY (PropertyId)
-    REFERENCES FileProperty(PropertyId);
+--ALTER TABLE FileMetadata
```

Comment:
I think so. I did this much earlier on as this code was causing an issue when testing; I think it was due to the fact that PropertyId is a foreign key belonging to FileProperty. It might be it's easier to do this now since FileProperty is joined. I'll try it.